clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/provider/view/EcsClusterProvider.java

```diff
@@ -36,4 +51,54 @@ public EcsClusterProvider(Cache cacheView) {
   public Collection<EcsCluster> getAllEcsClusters() {
     return ecsClusterCacheClient.getAll();
   }
+
+  // TODO include[] input of Describe Cluster is not part of this implementation, need to implement
+  // in the future.
```

Comment:
nit: as phrased, makes it sound like we're missing something we intend to add, when it's really that we don't need the additional info yet. Suggest amending to say "...in the future **if additional properties are needed**"

Reply:
make sense.